{"uid":"2cf054536c0acd69","name":"test_broken","fullName":"tests.pytest_doc.how_to.test_status#test_broken","historyId":"7e64cee7709f45cb74575ac5446607e3","time":{"start":1751521220939,"stop":1751521220939,"duration":0},"status":"broken","statusMessage":"Exception: This test is broken","statusTrace":"def test_broken():\n>       raise Exception(\"This test is broken\")\nE       Exception: This test is broken\n\ntests/pytest_doc/how_to/test_status.py:20: Exception","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"parentSuite","value":"tests.pytest_doc.how_to"},{"name":"suite","value":"test_status"},{"name":"host","value":"pkrvmbietmlfzoi"},{"name":"thread","value":"2425-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.pytest_doc.how_to.test_status"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"2cf054536c0acd69.json","parameterValues":[]}